packages/documentation/components/icon/icon.doc.md

```diff
@@ -20,16 +20,13 @@ TODO: Replace with real-world example from /demos
 
 ```vue
 <script setup>
-  import { AcvIcon } from '@acronis-platform/vue';
-  // or
-  import Icon from '@acronis-platform/components/icon/Icon.vue';
+import { AcvIcon } from "@acronis-platform/ui-component-library";
```

Comment:
You have rollback these changes. We have a different prettify config in docgen.

Reply:
need to fix it, because it's hard to revert this file every time